substrate-node/pallets/pallet-smart-contract/src/cost.rs

```diff
@@ -244,10 +244,15 @@ pub fn calculate_cost_in_tft_from_musd<T: Config>(
     total_cost: u64,
 ) -> Result<u64, DispatchErrorWithPostInfo> {
     let avg_tft_price = pallet_tft_price::AverageTftPrice::<T>::get();
-    ensure!(avg_tft_price > 0, Error::<T>::TFTPriceValueError);
+    let min_tft_price = pallet_tft_price::MinTftPrice::<T>::get();
+
+    let mut tft_price = avg_tft_price;
```

Comment:
could be changed to:
let tft_price = avg_tft_price.max(min_tft_price);
or 
let tft_price = min_tft_price.max(avg_tft_price);